src/main/java/org/skriptlang/skript/bukkit/particles/elements/expressions/ExprGameEffectWithData.java

```diff
@@ -0,0 +1,106 @@
+package org.skriptlang.skript.bukkit.particles.elements.expressions;
+
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.util.Patterns;
+import ch.njol.util.Kleenean;
+import org.bukkit.Effect;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.addon.AddonModule.ModuleOrigin;
+import org.skriptlang.skript.bukkit.particles.GameEffect;
+import org.skriptlang.skript.bukkit.particles.registration.DataGameEffects;
+import org.skriptlang.skript.bukkit.particles.registration.EffectInfo;
+import org.skriptlang.skript.registration.SyntaxInfo;
+import org.skriptlang.skript.registration.SyntaxRegistry;
+
+import static org.skriptlang.skript.registration.DefaultSyntaxInfos.Expression.builder;
```

Comment:
any reason for the import?

Reply:
it was ungainly

Reply:
I think `SyntaxInfo.Expression.builder` should work